### PR TITLE
[IGA] Support Nitsche Condition Test

### PR DESCRIPTION
**Pull request by rickyaristio**
**Description**
This PR adds the single-patch tests for the support Nitsche conditions. This is the part of #8858 (support condition tests), which has a dependency on #8866 (support Nitsche condition).

### CODE REVIEW

**Comment by tteschemacher** [comment]
The tests look fine. Let's wait until the other branches are merged!

File: applications/IgaApplication/tests/test_IgaApplication.py
Comment (tteschemacher) [inline applications/IgaApplication/tests/test_IgaApplication.py:77]: 👍

**Review by tteschemacher — APPROVED** [review]
Awesome! Thanks and well done!

File: applications/IgaApplication/tests/test_IgaApplication.py
Comment (rickyaristio) [reply applications/IgaApplication/tests/test_IgaApplication.py:77]: Yeah, we need this as FEAST doesn't work on Windows OS. The next step is to figure out how to run the Nitsche condition on Windows, perhaps try/use another solver. I'll save it for later.

**Pull request merged**
